bindings/matrix-sdk-ffi/src/timeline.rs

```diff
@@ -611,6 +632,12 @@ impl From<&matrix_sdk::ruma::events::room::ImageInfo> for ImageInfo {
     }
 }
 
+impl From<&matrix_sdk::ruma::events::room::message::AudioInfo> for AudioInfo {
+    fn from(info: &matrix_sdk::ruma::events::room::message::AudioInfo) -> Self {
+        Self { duration: info.duration.map(|d| d.as_secs()), size: info.size.map(Into::into) }
```

Comment:
On the protocol level, the duration is actually in milliseconds. But we don't have to drop down to a bare number, since UniFFI supports `Duration` as a built-in type: https://mozilla.github.io/uniffi-rs/udl/builtin_types.html

Reply:
Actually it seems like the UniFFI proc-macro API doesn't support that yet. In that case, can you switch to `as_millis` here and add a FIXME comment to the field declaration that it should be `std::time::Duration` in the future, once supported?